ui/src/app/kvm/views/KVMList/KVMListHeader/KVMActionFormWrapper/KVMActionFormWrapper.test.tsx

```diff
@@ -83,27 +80,6 @@ describe("KVMActionFormWrapper", () => {
         </MemoryRouter>
       </Provider>
     );
-    expect(wrapper.find("[data-test='kvm-action-form-wrapper']").exists()).toBe(
-      true
-    );
     expect(wrapper.find("RefreshForm").exists()).toBe(true);
   });
-
-  it("clears action form if no KVMs are selected", () => {
```

Comment:
I think this one needs to end up in KVMListHeader.test.tsx.

Reply:
Done!